node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
The [resource keys](https://github.com/Microsoft/vsts-task-lib/blob/master/node/lib.json#L3) themselves are prefixed with LIB_ as a means to avoid collisions with user resource keys.